### PR TITLE
test browser example compiles

### PR DESCRIPTION
**Pull request by charlierudolph**
The browser example had some errors building after some changes. 

Added a script for testing it and run that on CI.

### CODE REVIEW

File: src/formatter/get_color_fns.ts [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
 import Status from '../status'
+import { messages } from 'cucumber-messages'
 
 colors.enable()
 
 export type IColorFn = (text: string) => string
 
 export interface IColorFns {
-  forStatus(status: Status): IColorFn
+  forStatus(status: messages.TestResult.Status): IColorFn
```
[the remark this answers is not in the document]
Comment (charlierudolph) [reply src/formatter/get_color_fns.ts]: What was a blocker for? Using the re-exported Status as a enum appears fine (ex Status.PASSED), but using it as type appears to break the browser example.

If we don't re-export it, custom formatters / user code need to cucumber-messages to get access to the status enum. 

I'm interested if the changes in this PR would fix api extractor or not.

[the remark this answers is not in the document]
Comment (davidjgoss) [reply src/formatter/get_color_fns.ts]: I think it runs into some kind of edge case TypeScript bug from the combination for re-exporting from a separate package, and maybe it also being nested in namespaces. I talked to one of the rushstack folks a bit and filed https://github.com/microsoft/rushstack/issues/1667 but haven't had a chance to go back to it and really dig in yet.

If we were to remove the re-export `Status` from cucumber-js that would unblock the api-extractor piece, but with the downside you mentioned, although I think I would consider that mitigated if editor support was decent (i.e. if vscode, webstorm would smartly pick it up).

[the remark this answers is not in the document]
Comment (charlierudolph) [reply src/formatter/get_color_fns.ts]: Does this need to hold up this PR? Definitely needs to be solved but unsure it needs to block this. Or can we move this conversation to the typescript tasks issue?

[the remark this answers is not in the document]
Comment (davidjgoss) [reply src/formatter/get_color_fns.ts]: Sorry yeah, we can talk about it over in that ticket, no need to slow this one down.

**Pull request merged**